pkg/api/validation/dynakube/oneagent.go

```diff
@@ -60,7 +60,7 @@ func conflictingOneAgentConfiguration(_ context.Context, _ *Validator, dk *dynak
 }
 
 func conflictingOneAgentNodeSelector(ctx context.Context, dv *Validator, dk *dynakube.DynaKube) string {
```

Comment:
shouldn't we validate the conflicting node selectors in a single location and not for both oneagent and log agent in a slightly different way? We can't have 2 of them on the same node, no matter if it's log agent or oneagent

Reply:
before the PR it was separated, but now it's in 1 function, sooo 😅